test-data/unit/check-newsemanal.test

```diff
@@ -2179,6 +2179,12 @@ else:
     def f(x) -> None:
         y()  # E: "str" not callable
 
+[case testNewAnalyzerClassMemberRedefinition]
+class C:
+    def __init__(self) -> None:
+        self.foo = 12
+        self.foo: int = 12  # E: Name 'self.foo' already defined on line 3
+        
```

Comment:
I am not very sure about this. Should I really add same tests to new semantic analyzer tests?

Reply:
You don't need to duplicate tests, most test are currently run with both analyzers.

Reply:
Removed redundant tests.